src/main/java/teammates/ui/webapi/output/SessionResultsData.java

```diff
@@ -77,6 +78,30 @@ public SessionResultsData(FeedbackSessionResultsBundle bundle, StudentAttributes
         return questions;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Is this method added just for the sake of testing?

If yes, please convert it to helper method in the test file or you are responsible to write a comprehensive test for it (which I think it is not necessary).

Reply:
I see your point. I would like to convert it to a helper method, but here `QuestionOutput` is an inner class that has private access. Should I change the accessibility of the class or is there any wordaround?

Reply:
You can make the change.